api/services/Reddit.js

```diff
@@ -139,6 +139,12 @@ exports.sendPrivateMessage = function (refreshToken, subject, text, recipient, c
   makeRequest(refreshToken, 'POST', url, data, 25, callback);
 };
 
+exports.sendReply = function (refreshToken, text, parent_id, callback) {
```

Comment:
Oh, I was originally going to have it reply to previous modmails for a user rather than creating a new one every time, but I ended up dropping the idea. I guess I forgot to remove the Reddit.js function. (It should be functional anyway, so we may as well keep it in case we need it for something later.)